modules/cloud-credential-operator.adoc

```diff
@@ -8,8 +8,275 @@
 [discrete]
 == Purpose
 
-The Cloud Credential Operator manages cloud provider credentials as Kubernetes
-custom resource definitions (CRDs).
+The Cloud Credential Operator (CCO) manages cloud provider credentials as
+Kubernetes custom resource definitions (CRDs). The CCO syncs on
+`credentialsRequest` custom resources (CRs) to allow {product-title} components
+to request cloud provider credentials with the specific permissions that are
+required for the cluster to run.
+
+By setting different values for the `credentialsMode` parameter in the
+`install-config.yaml` file, the CCO can be configured to operate in several
+different modes. If no mode is specified, or the `credentialsMode` parameter is
+set to an empty string (`""`), the CCO operates in its default mode.
+
+[discrete]
+=== Default behavior
+For platforms where multiple modes are supported (AWS, Azure, and GCP), when the CCO operates in its default mode, it checks the provided credentials
```

Comment:
As an example, this line is soft-wrapped. The lines above and below it are hard-wrapped.

(Soft-wrap lines as seen from within GitHub run all the way to the edge of the browser window.)

Reply:
Yeah - asked about this Friday. I had been doing that myself because everything else was that way. The additions to this line came after that discussion when I found out I didn't have to do that any more :) Think I need to go back and remove hard wraps?

Reply:
@jeana-redhat, if it isn't too much work -- given how close docs freeze is, maybe it's okay for now.

Reply:
I'll save this one for a stretch goal. I also want to do some other things with this content outside of required work for 4.6 so I can take it as part of that improvement effort if I don't get to it :)